DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -116,8 +119,9 @@ public async Task<Variable<T>> VariableAsync<T>(User user, string key, T default
             }
             catch (DVCException e)
             {
-                if (e.IsRetryable())
-                {
+                if (e.IsUserError() && !e.Message.Contains("Feature not found for key")) {
```

Comment:
can't check the status code here?

Reply:
this also looks like the wrong message for this particular case, is that an API bug?

Reply:
the api responds 404 if either the sdk key or the variable key is not found, so this is the only way i could find to differentiate them. But yeah the message is kinda misleading, seems to be an api bug

Reply:
if the SDK is invalid it should response 401 unauthorized